SPECS/heimdal/heimdal.spec

```diff
@@ -0,0 +1,869 @@
+ln -s mech.5.gz %{buildroot}%{_mandir}/man5/qop.5.gz
+
+%find_lang %{name} --all-name
+
+%post server
+%if 0%{?_with_systemd}
+  %systemd_post heimdal-kdc.service
+  %systemd_post heimdal-ipropd-master.service
+  %systemd_post heimdal-ipropd-slave.service
+  %systemd_post heimdal-kadmind.service
+  %systemd_post heimdal-kpasswdd.service
+%else
+  /sbin/chkconfig --add heimdal-kdc
+  /sbin/chkconfig --add heimdal-ipropd-master
+  /sbin/chkconfig --add heimdal-ipropd-slave
+  /sbin/chkconfig --add heimdal-kadmind
+  /sbin/chkconfig --add heimdal-kpasswdd
+%endif
+
+%preun server
+%if 0%{?_with_systemd}
+  %systemd_preun heimdal-kdc.service
+  %systemd_preun heimdal-ipropd-master.service
+  %systemd_preun heimdal-ipropd-slave.service
+  %systemd_preun heimdal-kadmind.service
+  %systemd_preun heimdal-kpasswdd.service
+%else
+  if [ $1 -eq 0 ] ; then
+     /sbin/service heimdal-kdc stop >/dev/null 2>&1 || :
+     /sbin/chkconfig --del heimdal-kdc
+     /sbin/service heimdal-ipropd-master stop >/dev/null 2>&1 || :
+     /sbin/chkconfig --del heimdal-ipropd-master
+     /sbin/service heimdal-ipropd-slave stop >/dev/null 2>&1 || :
+     /sbin/chkconfig --del heimdal-ipropd-slave
+     /sbin/service heimdal-kadmind stop >/dev/null 2>&1 || :
+     /sbin/chkconfig --del heimdal-kadmind
+     /sbin/service heimdal-kpasswdd stop >/dev/null 2>&1 || :
+     /sbin/chkconfig --del >/dev/null
+  fi
+%endif
+
+%postun server
+%if 0%{?_with_systemd}
+  %systemd_postun heimdal-kdc.service
+  %systemd_postun heimdal-ipropd-master.service
+  %systemd_postun heimdal-ipropd-slave.service
+  %systemd_postun heimdal-kadmind.service
+  %systemd_postun heimdal-kpasswdd.service
+%else
+  if [ $1 -eq 1 ] ; then
+     /sbin/service heimdal-kdc condrestart >/dev/null 2>&1 || :
+     /sbin/service heimdal-ipropd-master condrestart >/dev/null 2>&1 || :
+     /sbin/service heimdal-ipropd-slave condrestart >/dev/null 2>&1 || :
+     /sbin/service heimdal-kadmind condrestart >/dev/null 2>&1 || :
+     /sbin/service heimdal-kpasswdd condrestart >/dev/null 2>&1 || :
+  fi
+%endif
+
+%if (0%{?rhel} && 0%{?rhel} < 8)
+%post libs
+/sbin/ldconfig
+/sbin/install-info %{_infodir}/%{name}.info %{_infodir}/dir || :
+
+%preun libs
+if [ $1 = 0 ] ; then
+  /sbin/install-info --delete %{_infodir}/%{name}.info %{_infodir}/dir || :
+fi
+
+%postun libs -p /sbin/ldconfig
+%endif
+
+%files libs -f %{name}.lang
+%dir %{bindir}
+%dir %{bindir}/bin
+%dir %{libdir}
+%config(noreplace) %{_sysconfdir}/ld.so.conf.d/%{name}-%{_arch}.conf
+%{libdir}/lib*.so*
+%{libdir}/windc.so*
+%{_infodir}/heimdal.info*
+%{_infodir}/hx509.info*
+%{_mandir}/man5/%{name}-krb5.conf.5*
+%{_mandir}/man5/qop.5*
+%{_mandir}/man5/mech.5*
+%{_mandir}/man8/kerberos.8*
+%{_bindir}/string2key
+%{_mandir}/man8/string2key.8*
+%{_libexecdir}/kdigest
+%{_mandir}/man8/kdigest.8*
+%{_bindir}/verify_krb5_conf
+%{_mandir}/man8/verify_krb5_conf.8*
+%{_libexecdir}/digest-service
+%doc %{_pkgdocdir}
+%license LICENSE
```

Comment:
Nit: could you move these two to the beginning of the file list? That's where we tend to keep them in other specs and it's easier to check if we've got the license right.